src/mmcache.hpp

```diff
@@ -88,6 +88,7 @@ class mm_cache {
       cache[i].finger_print_cnt_sum = 0;
       cache[i].activated = 0;
     }
+    memset(head_mm, 0, sizeof(uint64_t) * HEAD_MM_ARRAY_SIZE);
```

Comment:
Can you explain if this will affect the mapping results?

Reply:
This does not. It removes some warnings about initialization.